docs/DeveloperGuide.md

```diff
@@ -735,28 +739,105 @@ Priorities: High (Must-Have) - * * *, Medium (Good-To-Have) - * *, Low (To-Forgo
   
 **MSS**
 
-1.  User requests to delete a specific patient in the list.
-2.  MediConnect deletes the person.
+1.  User requests to delete a specific patient in the system.
+2.  MediConnect deletes the patient.
+3.  MediConnect shows a success message to the user.
 
     Use case ends.
 
 **Extensions**
 
-* 1a. MediConnect detects that the list is empty.
+* 1a. MediConnect detects that the patient list is empty.
 
   Use case ends.
 * 1b. MediConnect detects that the given index is invalid.
     * 1b1. MediConnect shows an error message.
     * 1b2. MediConnect requests for the correct input.
 
-      Use case resumes at step 2.
+      Use case resumes at step 1.
+
+**Use case: UC06 Edit a specified patient’s remark**
+
+**MSS**
+
+1.  User requests to edit a patient’s remarks.
+2.  MediConnect edits the patient’s remarks.
+3.  MediConnect shows a success message to the User.
+
+Use case ends.
+
+**Extensions**
+
+* 1a. MediConnect detects that the given command is invalid.
+    * 1a1. MediConnect shows an error message.
+    * 1a2. MediConnect requests the correct input
+
+      Use case resumes at step 1.
+* 1b. MediConnect detects that the given patient does not exist in the system.
```

Comment:
is this supposed to be index is invalid?

Reply:
i think this is fine? Index invalid would be more specific?

Reply:
would that be too specific and define the design? hmm

Reply:
ok